src/mf/mf-hid.cpp

```diff
@@ -37,6 +38,8 @@ namespace librealsense
 {
     namespace platform
     {
+        using namespace utilities::string;
```

Comment:
Fix it in specific places, or use a more specific using clause. Try to never do a general using statement please.

Reply:
done